packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -331,7 +332,7 @@ class Route extends EmberObject {
     @method _reset
     @since 1.7.0
   */
-  _reset(isExiting: boolean, transition: Transition) {
+  reset(isExiting: boolean, transition: Transition) {
```

Comment:
Seems like this is still private, why is the underscore removed? Also, shouldn't we be worried about userland naming collisions?